<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>508</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_PVP">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>84</y>
         <width>76</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>人人对战</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_PVC">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>76</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>人机对战</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_CVC">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>298</y>
         <width>76</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>机机对战</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>115</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>先手</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>115</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>后手</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>地址：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="addressLineEdit">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="cancelButton_2">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="portlineEdit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>对话：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
